auth-web/src/models/business.ts

```diff
@@ -14,8 +14,10 @@ export interface FolioNumberload {
 }
 
 export interface CorpType {
-    code: string
+    code: string // may be corp type or TMP / NR / RTMP
+    default?: boolean
     desc: string
+    legalType: CorpTypes // actual corp type
```

Comment:
@vysakh-menon-aot This is the new property I propose in this sub-object (in get-affiliations response).